sentry-ruby/lib/sentry/rake.rb

```diff
@@ -23,7 +23,7 @@ def execute(args=nil)
       return orig_execute unless Sentry.initialized? && Sentry.get_current_hub
```

Comment:
`orig_execute` should receive the argument too

Reply:
Thanks, I fixed!